.golangci.yml

```diff
@@ -3,6 +3,8 @@
 
 run:
   timeout: 3m
+  skip-dirs:
+    - cmd/envtool/internal/testmatch # due to files match.go and match_test.go from the Go standard library
```

Comment:
Why do we need that change?

Reply:
I've removed it (it was leftover from my first submitted change), but here is the context why it was there in the first place:

This change was needed because files files match.go and match_test.go (5ff7190927a25e30f619279b5e5078b98c7cbb9a) were imported from the standard testing library and the linter is incompatible with it, and maintaining it 'as is' would be much easier than adapting the code to satisfy the linter configuration.